lms/renderers.py

```diff
@@ -0,0 +1,23 @@
+from datetime import UTC, datetime
+
+from pyramid.renderers import JSON
+
+
+def json_iso_utc():
+    """Return a JSON renderer that formats dates as `isoformat`.
+
+    This renderer assumes datetimes without tz info are in UTC and
+    includes that in the datetime objects so the resulting string
+    includes tz information.
+    """
+
+    renderer = JSON()
+
+    def _datetime_adapter(obj: datetime, _request) -> str:
```

Comment:
From https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/renderers.html#using-the-add-adapter-method-of-a-custom-json-renderer